package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "react-numerics",
+  "main": "./dist/react-numerics.umd",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/",
+    "libs/react-numerics/src",
```

Comment:
`libs/` and `apps/` are folder from NX, they aren't needed now, can pull out src to the root of the project and remove those? You will also need to update the entry point in the vite config as well

in a similar vein, we have two READMEs (one at root and one in src) we might wanna combine them